content/30min-setup.md

```diff
@@ -49,9 +49,12 @@ back instance:
 $ ./taiga-manage.sh [COMMAND]
 ```
 
-Default access for the application is **http://localhost:9000**.
+Default access for the application is **http://localhost:9000** 
```

Comment:
If you're testing it in your own machine, you can access the application in **http://localhost:9000** as it's by default. If you're deploying in a server, you'll need to configure hosts and nginx as described later.

Reply:
Ok, it sounds good enough to me. I'll update the three PRs accordingly.
Thanks